build/cli.ts

```diff
@@ -228,10 +228,10 @@ async function buildDocuments(
       fs.writeFileSync(liveSamplePath, html);
     }
 
-    for (const filePath of fileAttachments) {
+    for (const [basename, filePath] of fileAttachments) {
```

Comment:
This variable, coming from `buildDocumentInteractive` should also be called `fileAttachmentMap`:

```suggestion
    for (const [basename, filePath] of fileAttachmentMap) {
```